internal/generate/olm-catalog/csv.go

```diff
@@ -71,6 +72,9 @@ type BundleGenerator struct {
 	// toBundleDir is the bundle directory filepath where the CSV will be generated
 	// This is set according to the generator's OutputDir
 	toBundleDir string
+	// Subcommand includes the list of csv metadata fields which the user
+	// provides to the interactive prompts which appear while generating csv.
+	subcommand InteractiveCSVCmd
```

Comment:
```suggestion
	interactiveCSVCmd InteractiveCSVCmd
```

Keep the same name would not make more intuitive the understatement here? Wdyt?